packages/shared/styles/components/organisms/SfSidebar.scss

```diff
@@ -1,6 +1,7 @@
 @import "../../helpers";
 .sf-sidebar {
   &__aside {
+    --scrollable-max-height: var(--sidebar-content-max-height, 600px);
```

Comment:
@AdamPawlinski we can't use the static size. I will try to find some solution and give you feedback later

Reply:
I removed this property and changed the way in which max scrollable height is calculated.